CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
 
 #### Enhancements
 
+* Private SwiftUI State property rule now also applies to StateObject.  
+  [mt00chikin](https://github.com/mt00chikin)
```

Comment:
Better to just mention `@StateObject` in your previous entry as it has not yet been released.